ironfish/src/network/peers/connections/webRtcConnection.ts

```diff
@@ -197,42 +197,18 @@ export class WebRtcConnection extends Connection {
     }
   }
 
-  send = (message: NetworkMessage): boolean => {
+  _send = (data: Buffer): boolean => {
     if (!this.datachannel) {
       return false
     }
 
-    if (this.shouldLogMessageType(message.type)) {
-      this.logger.debug(
-        `${colors.yellow('SEND')} ${this.displayName}: ${displayNetworkMessageType(
-          message.type,
-        )}`,
-      )
-    }
-
     if (!this.datachannel.isOpen()) {
       this.logger.debug('Datachannel no longer open, closing connection')
       this.close()
       return false
     }
 
-    const data = message.serializeWithMetadata()
-    try {
```

Comment:
I can't say for sure that this try/catch is no longer needed -- might be worth re-adding it unless you checked it out

Reply:
I hoisted it up into the send function: https://github.com/iron-fish/ironfish/pull/3590/files#diff-ed1400fde596eef381f9b8abb8faaa1965e28b481b5f87c829c95c16df06f918R139

Happy to move it back if we prefer - I don't feel strongly one way or the other, I think I moved it just to minimize the amount of differences between the `_send` functions

Reply:
Resolving this, I noticed you boosted it up to the connection level